src/usr/local/www/status_dhcp_leases.php

```diff
@@ -379,11 +379,12 @@ function remove_duplicate($array, $field) {
 					<th><?=gettext("IP address")?></th>
 					<th><?=gettext("MAC address")?></th>
 					<th><?=gettext("Hostname")?></th>
+					<th><?=gettext("Description")?></th>
 					<th><?=gettext("Start")?></th>
 					<th><?=gettext("End")?></th>
-					<th data-sortable="false"><?=gettext("Online")?></th>
-					<th data-sortable="false"><?=gettext("Lease Type")?></th>
-					<th data-sortable="false"><?=gettext("Actions")?></th>
```

Comment:
Don't think it really matters one way or the other.  Not much point in sorting on the actions column, but not a problem for it to be sortable either.  So either way is fine by me.  Thanks.